Ginger/Ginger/BusinessFlowPages_New/AddActionMenu/APINavPage.xaml.cs

```diff
@@ -70,9 +71,12 @@ public APINavPage(Context context)
 
         private void MContext_PropertyChanged(object sender, System.ComponentModel.PropertyChangedEventArgs e)
         {
-            if (e.PropertyName is nameof(mContext.Activity) || e.PropertyName is nameof(mContext.Target))
+            if (this.IsVisible && MainAddActionsNavigationPage.isPanelExpanded)
```

Comment:
check for this.IsVisible is not enough?

Reply:
No Meni. it's not enough. In case, panel is collapsed with the current page being visible, it'll still trigger as Page is visible inside the panel (IsVisible is true here even while panel is collapsed)

Reply:
ok , so keep it.